src/components/updated-terms-conds/index.njk

```diff
@@ -0,0 +1,149 @@
+{% extends "govuk/template.njk" %}
+{% from "govuk/components/cookie-banner/macro.njk" import govukCookieBanner %}
+{% from "govuk/components/phase-banner/macro.njk" import govukPhaseBanner %}
+
+<link rel="stylesheet" href="../../assets/css/css.css" />
+
+{% block head %}
+  <!--[if !IE 8]><!-->
+    <link href="/public/style.css" rel="stylesheet">
+  <!--<![endif]-->
+
+  {# For Internet Explorer 8, you need to compile specific stylesheet #}
+  {# see https://frontend.design-system.service.gov.uk/supporting-ie8/#support-internet-explorer-8 #}
+  <!--[if IE 8]>
+    <link href="/govuk-frontend/all-ie8.css" rel="stylesheet">
+  <![endif]-->
+
+  {# For older browsers to allow them to recognise HTML5 elements such as `<header>` #}
+  <!--[if lt IE 9]>
+    <script src="/html5-shiv/html5shiv.js"></script>
+  <![endif]-->
+
+{% endblock %}
+
+{% block pageTitle %}
+    {% if error or errors %}
+        {{ 'general.errorTitlePrefix' | translate }}
+        -
+    {% endif %}
+    {% if pageTitleName %}
+        {{ pageTitleName }}
+        -
+    {% endif %}
+    {{ 'general.serviceNameTitle' | translate }}
+{% endblock %}
+
+{% block bodyStart %}
+ {% include 'common/layout/banner.njk' %}
+{% endblock %}
+
+{% block header %}
+  {{ govukHeader({
+  homepageUrl: "#",
+  serviceUrl: "#"
+}) }}
+{% endblock %}
+
+{% block main %}
+      <div class="govuk-width-container {{ containerClasses }}">
+        {{ govukPhaseBanner({
+          tag: {
+            text: 'general.phaseBanner.tag' | translate
+          },
+          html: 'This is a new service – your <a class="govuk-link" href="#">feedback</a> will help us to improve it.'
+        }) }}
+	      {% block beforeContent %}{% endblock %}
+	      <main class="govuk-main-wrapper {{ mainClasses }}" id="main-content" role="main"{% if mainLang %} lang="{{ mainLang }}"{% endif %}>
+          <div class="govuk-grid-row interrupt-screen">
+            <div class="govuk-grid-column-two-thirds">
+              {% extends "common/layout/base.njk" %}
```

Comment:
how come this page is being imported with the base layout inside the main content block?